engine/core/src/main/java/org/datacleaner/job/builder/AnalysisJobBuilder.java

```diff
@@ -1205,4 +1219,37 @@ public Boolean eval(InputColumn<?> inputColumn) {
 
         return result;
     }
+
+    public AnalysisJobBuilder getTopLevelJobBuilder(){
```

Comment:
In terms of naming I kinda like the term "ultimate parent" rather than "top level". Maybe just me, but a suggestion at least.